services/web/server/src/simcore_service_webserver/users/_notifications.py

```diff
@@ -106,6 +108,8 @@ class Config:
                     "date": "2023-02-23T16:28:13.122Z",
                     "product": "s4l",
                     "read": False,
+                    "resource_id": "3fb96d89-ff5d-4d27-b5aa-d20d46e20e12",
+                    "user_from_id": "2",
```

Comment:
Q: what about existing notifications which do not have this field set. Can we have an example that is still validated against one such model?

Reply:
You previous 3 examples do not have these fields 10 lines above...